packages/commonwealth/server/migrations/20250411211930-add-community-namespace-verified.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.addColumn('Communities', 'namespace_verified', {
```

Comment:
Combine with above? Both require a full table lock anyways so might as well just grab it once.